### PR TITLE
[MIG] partner_capital: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
- Minor migration tweaks

cc @Tecnativa

### CODE REVIEW

**Review by yajo — APPROVED** [review]
Please squash all my commits in 1, all Transbot's + en.po removal in 1, and yours in 1. 3 in total.

File: partner_capital/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Partners Capital
+================
+
+This module extends the functionality of partners to support setting the
+country of origin of capital, registered capital amount and turnover.
```
Comment (angelmoya) [inline partner_capital/README.rst]: On code "registered capital" was renamed as "Capital amount", maybe could be better to update also in README

**Review by angelmoya — APPROVED** [review]
Small remark, but it's OK

File: partner_capital/README.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+partner_capital%0Aversion:%20
+10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Images
+------
+
+* `Antiun Ingeniería S.L. <http://www.antiun.com>`_: Icon.
```
Comment (rafaelbn) [inline partner_capital/README.rst]: Change / fix

File: partner_capital/README.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Contributors
+------------
+
+* Antonio Espinosa <antonioea@antiun.com>
```
Comment (rafaelbn) [inline partner_capital/README.rst]: antonio.espinosa@tecnativa.com

File: partner_capital/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# © 2015 Antiun Ingeniería S.L. - Antonio Espinosa
+# © 2015 Tecnativa S.L. - Jairo Llopis
+# © 2017 Tecnativa S.L. - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Partners Capital',
+    'version': '10.0.1.0.0',
+    'category': 'Customer Relationship Management',
+    'author': 'Antiun Ingeniería S.L.',
```
Comment (rafaelbn) [inline partner_capital/__manifest__.py]: Tecnativa + Add OCA as in templates

File: partner_capital/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    'website': 'http://www.antiun.com',
```
Comment (rafaelbn) [inline partner_capital/__manifest__.py]: www.tecnativa.com

**Review by rafaelbn — CHANGES_REQUESTED** [review]

**Comment by chienandalu** [comment]
@rafaelbn Please review to rebase and merge

**Review by luismontalba — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**